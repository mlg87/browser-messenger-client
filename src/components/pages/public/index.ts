export { default as Landing } from './Landing/Landing';
export { default as Login } from './Login/Login';
export { default as NotFound } from './NotFound/NotFound';
export { default as Register } from './Register/Register';
