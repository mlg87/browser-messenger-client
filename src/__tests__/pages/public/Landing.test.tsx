import { render } from '@testing-library/react';
import { Provider } from 'mobx-react';
import React from 'react';
import { MemoryRouter } from 'react-router';
import { Landing } from '../../../components/pages/public';
import appState from '../../../stores';

const setup = () => render(
    <Provider {...appState}>
        <MemoryRouter>
            <Landing />
        </MemoryRouter>
    </Provider>
);

describe('Landing', () => {

    it('renders', () => {

        const { container } = setup();
        expect(container.firstChild).toMatchSnapshot();

    });

});
