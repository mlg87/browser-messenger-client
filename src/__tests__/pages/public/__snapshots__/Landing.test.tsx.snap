// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing renders 1`] = `
<div
  class="container"
>
  <div>
    <h1>
      welcome. let's do some chatting.
    </h1>
    <span
      class="link-container"
    >
      <a
        href="/login"
      >
        login
      </a>
      <a
        href="/register"
      >
        register
      </a>
    </span>
  </div>
</div>
`;
